src/item.h

```diff
@@ -1268,6 +1268,7 @@ class item : public visitable<item>
          */
         /*@{*/
         void set_var( const std::string &name, int value );
+        void set_var( const std::string &name, std::int_fast64_t value );
```

Comment:
```suggestion
        void set_var( const std::string &name, long long value );
```
You can't overload this on `int_fast64_t` because it's sometimes the same as `long` (and sometimes not).